client/src/SalaryBarGraph.js

```diff
@@ -0,0 +1,156 @@
+import _ from 'lodash'
+import d3 from 'd3'
+
+const SALARY_CAP = 9999999999999
```

Comment:
if I set this and uncomment some code I can draw horizontal lines on the league graph and change the color of any players crossing it to yellow. This isn't super useful though so I may just delete it.